### DISCUSSION TITLE
Android App?

### DISCUSSION THREAD

**Discussion by Doni49** [hassio-addons/addon-bitwarden#257] [General]
When I first started considering Vaultwarden, I watched a YT video talking about VW is actually Bitwarden under another name and he said that it works with the bitwarden android app.

I have VW installed on my HA server.  After installing it, I was able to browse to the server (using my duckdns domain and encryption certificate from LetsEncrypt) and create an account for myself.

Having done that, I installed bitwarden on my android phone and tried to connect it to the server. When I enter my email address (the one I used when I created my account), BW throws an error message that says “We were unable to process your request. Please try again or contact us.”

I asked about this in the bitwarden forum and go this response:
`Vaultwarden is a third-party app that is not affiliated with Bitwarden in any way, and is not guaranteed to be compatible with official Bitwarden products. Your best bet is to turn to the Vaultwarden community for support.`

Does the bitwarden app NOT work with VW?  If it does, what do I need to do to use it?

**Comment by beMoD** [comment]
Have you figured this out yet? Did you change the app to use your server?

**Comment by KINGLIFER** [comment]
Let us know what you found out. Thanks.
